crates/ruma-common/src/identifiers/client_secret.rs

```diff
@@ -20,18 +23,10 @@ impl ClientSecret {
     #[cfg(feature = "rand")]
     pub fn new() -> Box<Self> {
```

Comment:
Shouldn't these become `Owned`?

Reply:
Yes, this PR isn't finished yet (see todo list at the top).